src/include/reconcile.h

```diff
@@ -66,6 +72,17 @@ struct __wt_rec_chunk {
     uint32_t auxentries;
 };
 
+#define WT_REC_CHUNK_TA_UPDATE(session, chunk, tw)                                    \
+    do {                                                                              \
+        WT_TIME_AGGREGATE_UPDATE((session), &(chunk)->ta, (tw));                      \
+        WT_TIME_AGGREGATE_UPDATE((session), &(chunk)->ta_after_split_boundary, (tw)); \
+    } while (0)
+#define WT_REC_CHUNK_TA_MERGE(session, chunk, tw)                                    \
```

Comment:
Because of macro substitution rules I renamed it to `agg` the `chunk->ta` var was getting substituted with whatever was in `ta`. Does `agg` work for you?

Reply:
Yes, that works for me.